merge_state_block/release.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
On a similar note, we have previously said that block files should follow a specific naming structure: Take the ClassName and turn it into class_name and add _block.py to the end.

There's no technical reason for this to exist but it is a naming convention that we had agreed upon.